src/week2/binarySearch/BinarySearch.java

```diff
@@ -0,0 +1,25 @@
+package week2.binarySearch;
+
+public class BinarySearch {
+    int n;
+    static final int notAPerfectSquare = -1;
```

Comment:
wrong naming convention

Reply:
hemant, can you please tell me whats wrong here